docs/modules/ROOT/pages/how-tos/connect-to-servala-control-cluster.adoc

```diff
@@ -0,0 +1,5 @@
+= Connect to a Servala control cluster
+
+Servala control clusters are vclusters that run within specific service clusters. This means they differ from service cluster such that they are not OpenShift clusters but rather vanilla Kubernetes.
+
+Their authentication is handled via the `kubelogin` plugin. For more details how to install and connect to the vcluster see: https://hub.syn.tools/vcluster/how-tos/oidc.html#_optional_setup_kubelogin[here].
```

Comment:
I am stuck in that page, what keycloak should I connect to? It's easier for me to just use the kubeconfig saved in vault

Reply:
We don't have to save it in vault. I can actually just add an example to the page and note down the fields that need to be adjusted.

Reply:
Can't we just publish the correct kubeconfig for all control-plane clusters like we do for the lieutenant-clusters? It's not like there is going to be a huge amount of those clusters

Reply:
I think the adjustment made is good enough for now.